cmd/config.go

```diff
@@ -251,19 +252,66 @@ func (d *ConfigDuration) UnmarshalYAML(unmarshal func(interface{}) error) error
 
 // GRPCClientConfig contains the information needed to talk to the gRPC service
 type GRPCClientConfig struct {
-	// ServerAddress is a single host:port combination that the gRPC client
-	// will, if necessary, resolve via DNS and then connect to. This field
-	// cannot be used in combination with `ServerIPAddresses` field.
+	// DNSAuthority is the server address to dial for gRPC DNS lookups. This
+	// field is optional and can only specified along with `ServiceDomain` and
+	// `ServerAddress`.
```

Comment:
Just noticed this is a little underspecified, I'll fix the language up tomorrow.